tests/test_smoke.py

```diff
@@ -1317,6 +1317,26 @@ def test_scp_logs():
     run_one_test(test)
 
 
+@pytest.mark.kubernetes
+def test_symlink_latest_logs():
+    test = Test(
+        'symlink_latest_logs',
+        [
+            'sky launch -c test1 --cloud kubernetes -- echo hi > log1.txt || true',
```

Comment:
nit - we could use `--cloud {generic_cloud}` here like we do in the rest of tests.

Reply:
Should I remove the mark for this test then so it can run on all clouds?

Reply:
Yes, if we use generic cloud we should remove the mark for Kubernetes.